### ISSUE TITLE
Invalid status flag when a hash message is empty

### ISSUE THREAD

**Issue by ekawat** [GMUCERG/LWC#32]
The following is generated for pdi.txt when a message is empty.

```
#### Hash                                                                                                                                                                                                                                     
#### MsgID=  3, KeyID=  0 Ad Size =    0, Hash_Tag Size =   32                                                                                                                                                                                
# Instruction: Opcode=Hash                                                                                                                                                                                                                    
INS = 80000000                                                                                                                                                                                                                                
# Info :                     Hash, EOI=0 EOT=1, Last=1, Length=0 bytes                                                                                                                                                                        
HDR = 73000000          
```

Instead, it should be as follow:

```
#### Hash                                                                                                                                                                                                                                     
#### MsgID=  3, KeyID=  0 Ad Size =    0, Hash_Tag Size =   32                                                                                                                                                                                
# Instruction: Opcode=Hash                                                                                                                                                                                                                    
INS = 80000000                                                                                                                                                                                                                                
# Info :                     Hash, EOI=1 EOT=1, Last=1, Length=0 bytes                                                                                                                                                                        
HDR = 77000000         
```

**Comment by kammoh** [comment]
Can we close this since #33 is in?

**Comment by kammoh** [comment]
I mark it as fixed as it seems to be covered by #33 please feel free to reopen if any issues persist.
